client/src/GachaContainer.jsx

```diff
@@ -1,45 +1,40 @@
-import React, { Component } from 'react';
-import { Segment, Button } from 'semantic-ui-react';
-
-import Summon from './components/Summon';
-import Servant from './components/Servant';
+import React, { Component } from "react";
+import { Segment, CardGroup } from "semantic-ui-react";
+import Summon from "./components/Summon";
+import Servant from "./components/Servant";
 
 class GachaContainer extends Component {
-    state = {
-        summoned: false,
-        loading: false,
-        servant: {}
-    };
+  state = {
+    summoned: false,
+    loading: false,
+    servant: {},
+    number: 0
```

Comment:
This needs validation.
I tried entering -1 and the app crashed.

Reply:
Noted. Will make necessary changes.